src/main/resources/site/processors/add-metadata.js

```diff
@@ -9,134 +9,137 @@ const view = resolve("add-metadata.html");
 
 exports.responseProcessor = function (req, res) {
     const site = libs.portal.getSite();
+    const siteConfig = libs.portal.getSiteConfig();
     const content = libs.portal.getContent();
-    const siteConfig = libs.common.getTheConfig(site);
 
-    const isFrontpage = site._path === content._path;
-    const pageTitle = libs.common.getPageTitle(content, site);
-    const titleAppendix = libs.common.getAppendix(site, isFrontpage);
+    /** We don't want 500 on controllers without content */
```

Comment:
Not sure, this comment is relevant for future code readers

Reply:
Since this method is used on content 99% of the time. I was thinking it was relevant since this was a bug we had.